### PR TITLE
HOSTEDCP-1005: Move Arm NodePool test into nodepool_test.go

### PR DESCRIPTION
**Pull request by bryan-cox**
**What this PR does / why we need it**:
Moves Arm NodePool test into nodepool_test.go

**Which issue(s) this PR fixes**:
Fixes [HOSTEDCP-1005](https://issues.redhat.com/browse/HOSTEDCP-1005)

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [x] Relevant issues have been referenced.
- N/A This change includes docs. 
- N/A This change includes unit tests.

### CODE REVIEW

**Comment by bryan-cox** [comment]
/retitle HOSTEDCP-1005: Move Arm NodePool test into nodepool_test.go

**Comment by bryan-cox** [comment]
test/bin error
/test e2e-aws

File: test/e2e/nodepool_arm.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+func (a *ArmNodepoolTest) Setup(t *testing.T) {
+	if globalOpts.Platform != hyperv1.AWSPlatform {
+		t.Skip("test only supported on platform AWS")
+	}
+
+	t.Log("Starting test ArmNodepoolTest")
+
+	clusterOpts.Arch = "arm64"
```
Comment (muraee) [inline test/e2e/nodepool_arm.go:37]: this doesn't have any effect as the cluster has already been created.

File: test/e2e/nodepool_arm.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+func (a *ArmNodepoolTest) BuildNodePoolManifest(defaultNodepool hyperv1.NodePool) (*hyperv1.NodePool, error) {
+	nodePool := &hyperv1.NodePool{
+		ObjectMeta: v1.ObjectMeta{
+			Name:      a.hostedCluster.Name + "-" + "test-arm-nodepool",
+			Namespace: a.hostedCluster.Namespace,
+		},
+	}
+	defaultNodepool.Spec.DeepCopyInto(&nodePool.Spec)
+
+	nodePool.Spec.Replicas = &oneReplicas
```
Comment (muraee) [inline test/e2e/nodepool_arm.go:49]: you would want to set the `arch` here for the new nodepool:
```
nodePool.Spec.Arch = "arm64"
```

**Comment by bryan-cox** [comment]
Tested this locally and it worked, thought Amazon was being slow.
/test e2e-aws

**Comment by bryan-cox** [comment]
This is not possible atm since the nodepool_test.go isn't utilizing a multi-arch image for the hosted cluster. Closing PR for now.

**Pull request closed without merging**